lib/cc/engine/analyzers/python/parser.rb

```diff
@@ -0,0 +1,57 @@
+require 'posix/spawn'
+require 'timeout'
+require 'json'
+
+module CC
+  module Engine
+    module Analyzers
+      module Python
+        class Parser
+          attr_reader :code, :filename, :syntax_tree
+
+          def initialize(code, filename)
+            @code = code
+            @filename = filename
+          end
+
+          def parse
+            runner = CommandLineRunner.new(python_env, self)
+            runner.run(code) do |ast|
+              json_ast = JSON.parse(ast)
+              @syntax_tree = json_ast
+            end
+
+            self
+          end
+
+          def python_env
```

Comment:
Hmm...does this method name accurately describes this? Seems more like a command to me.